assets/src/edit-story/app/index.js

```diff
@@ -47,6 +47,9 @@ import { useSnackbar, SnackbarProvider } from './snackbar';
 import Layout from './layout';
 import { Media3pApiProvider } from './media/media3p/api';
 
+/**
+ *
+ */
```

Comment:
Hmm, looks like all the JSDoc comments were added automatically by the local linter. 🤔

Reply:
Most warnings should go away with a rebase